### DISCUSSION TITLE
Import MY_SOTA_REF and STATION_CALLSIGN from ADIF

### DISCUSSION THREAD

**Discussion by hb9hnt** [magicbug/Cloudlog#1858] [Ideas / Feature Requests]
I'm not sure this is a feature request or just me not understanding how things work. At the moment I have two ways to log SOTA activations, either on paper or on my phone using [tSotaLog](https://codeberg.org/hb9hnt/tSotaLog/releases) (my logging app of which I'm most likely the only user...).

It's clear to me how I enter the paper log into Cloudlog to then create an export for sotadata: Create a new station location for the summit, enter QSOs for that station and export the log of this station.

When I use my app though, I end up with a ADIF file I would like to upload to sotadata and to Cloudlog. If I understand this correctly I would still have to create a station location on Cloudlog manually, even though the information required is contained in the `MY_SOTA_REF` and `STATION_CALLSIGN` fields in the ADIF.  If I do two SOTA activations on the same day things get even more difficult. I end up with a ADIF containing both activations, which is ok for sotadata. But for Cloudlog I would have to split the file for the two different locations - unless I missed something.

So I guess my question is: Is it possible to import this data from the ADIF without the unnecessary step of creating a station? If not, consider this a feature request. ;)

73, Beni HB9HNT

**Comment by magicbug** [comment]
Currently no, but its on the todo list.
